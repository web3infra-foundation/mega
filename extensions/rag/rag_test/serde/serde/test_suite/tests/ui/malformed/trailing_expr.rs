use serde_derive::Serialize;

#[derive(Serialize)]
struct S {
    #[serde(skip_serializing_if, x.is_empty())]
    x: Vec<()>,
}

fn main() {}
