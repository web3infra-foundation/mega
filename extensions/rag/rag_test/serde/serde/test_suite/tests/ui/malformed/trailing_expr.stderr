error: expected `=`
 --> tests/ui/malformed/trailing_expr.rs:5:32
  |
5 |     #[serde(skip_serializing_if, x.is_empty())]
  |                                ^
